test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -3423,6 +3825,47 @@ def test_cpu_controller_enforce_default_zero_job(self):
               zero_cpus_shares_fraction
               zero_cpus_quota_fraction
         """
+        root_quota_host1 = None
+        try:
+            root_quota_host1_str = \
+                self.du.run_cmd(hosts=self.hosts_list[0],
+                                cmd=['cat',
+                                     '/sys/fs/cgroup/cpu/cpu.cfs_quota_us'])
```

Comment:
These look like repeating blocks of statements that are best to be put in a class variable.

Reply:
Yes, but frankly, I am now trying to focus on making things work, not refactoring it.

Reply:
Done